sql/src/main/java/io/crate/operation/reference/doc/lucene/CollectorContext.java

```diff
@@ -63,7 +62,8 @@ public int readerId() {
         return readerId;
     }
 
-    public byte relationId() {
+    public Byte relationId() {
+        assert relationId != null : "relationId must not be null";
```

Comment:
i think it would be better to keep the primitive byte because of boxing issues and create a subclass, which throws an exception